future.tex

```diff
@@ -0,0 +1,83 @@
+\subsection{Future Projections}
+
+\begin{frame}
+	\frametitle{Future Nuclear Projections}
+	From the World Nuclear Association \cite{world_nuclear_association_nuclear_2017}.
```

Comment:
State why you're replicating this table. This slide describes the deployment scheme for the reactors in the simulation and is based in reality. It's input to the simulation.